dcc-storage-client/src/main/java/org/icgc/dcc/storage/client/util/Formats.java

```diff
@@ -20,7 +20,7 @@
 public class Formats {
 
   public static String formatBytes(long bytes) {
-    int unit = 1000;
+    int unit = 1024;
```

Comment:
units reported as KB/MB but unit = 1000 is actually KiB/MiB. Or the other way around.